server/models/subscription.ts

```diff
@@ -173,6 +173,9 @@ export default (
             notification_data: JSON.stringify(notification_data)
           }
       );
+      if (msg && isChainEventData(notification_data)) {
+        msg.dynamic_template_data.notification.path = `https://commonwealth.im/${notification_data.chainEventType.chain}/notificationsList?id=${notification.id}`;
```

Comment:
Can you update this to ${SERVER_URL} instead of commonwealth.im. And then I think we should be all good

Reply:
resolved